mypy/nodes.py

```diff
@@ -2025,6 +2027,15 @@ def deserialize(cls, data: JsonDict) -> 'TypeInfo':
         return ti
 
 
+def namedtuple_type_info(tup: 'mypy.types.TupleType', *args) -> TypeInfo:
```

Comment:
`*args` won't be type checked. It would be nice if the arguments would be spelled out -- alternatively, add an explicit `Any` annotation.